src/03-modality-agnostic-files.md

```diff
@@ -28,6 +28,7 @@ Every dataset MUST include this file with the following fields:
 | EthicsApprovals    | OPTIONAL. List of ethics committee approvals of the research protocols and/or protocol identifiers.                                                                                                                                                                |
 | ReferencesAndLinks | OPTIONAL. List of references to publication that contain information on the dataset, or links.                                                                                                                                                                     |
 | DatasetDOI         | OPTIONAL. The Document Object Identifier of the dataset (not the corresponding paper).                                                                                                                                                                             |
+| Preregistration    | OPTIONAL.  URL or Document Object Identifier of the time-stamped pre-registration of the study that produced the dataset.                                                                                                                                          |
```

Comment:
DOIs can be made URIs by prepending `doi:` (or URLs by prepending `https://doi.org/`), so this puts the address of one URI type on par with a full URI. An implementing tool would basically have to say:

```
try:
    val = parseurl(preregistration)
except:
    val = parseuri("doi:" + preregistration)
```

It seems conceptually cleaner to permit any URI:

```suggestion
| Preregistration    | OPTIONAL. URI of a time-stamped preregistration of the study that produced the dataset.                                                                                                                                                                            |
```

I'm not sure if there's an argument for narrowing to URL.

I am also :+1: for adding an example URI to add to the `dataset_description.json`.

Reply:
a `study that produced the dataset` sounds weird, people produce datasets
suggest
`URL or DOI of a time-stamped preregistration of the study from which the dataset originates.`